### PR TITLE
[HNC-466] - Update all artifactory references in pentaho/actions-common

### PR DESCRIPTION
**Pull request by mayur-hitachivantara**
Update all artifactory references in the` pentaho/actions-common` repository.

Tests:- 
 Merge Workflow Test[ here](https://github.com/pentaho/pdi-plugins-ee/actions/runs/7449035581).
 Pull Request Workflow Test [here](https://github.com/pentaho/pdi-plugins-ee/actions/runs/7459361772).
 Bootstrap Image used in Merge and Pull request workflows built [here](https://github.com/pentaho/actions-common/actions/runs/7447688366).

### CODE REVIEW

File: .github/workflows/bootstrap-image.yml [hunk 1]
```diff
@@ -21,22 +21,33 @@ jobs:
...
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3  
+      
+      # Build and Push Image with docker buildx 
+      - name: Build and Push to Artifactory
         id: build-image
-        run: docker build -t "${{ env.ARTIFACTORY }}/${{ env.IMAGE }}" -f .github/bootstrap-image/Dockerfile .
+        run: docker buildx build -t ${{ env.ARTIFACTORY }}/${{ env.IMAGE }} --push -f .github/bootstrap-image/Dockerfile --secret id=myuser,env=ARTIFACTORY_USER --secret id=mytoken,env=ARTIFACTORY_API_KEY .
+  
+      # - name: Build
```
Comment (cardosov) [inline .github/workflows/bootstrap-image.yml:36]: No need to keep commented code around.

File: settings.xml
Comment (cardosov) [inline settings.xml:1]: You can leave just the `pentaho.resolve.repo` server config inside of _servers_ list.

**Review by cardosov — CHANGES_REQUESTED** [review]
Haven't seen any build examples that are deploying

File: .github/workflows/bootstrap-image.yml (hunk 1 above)
Comment (mayur-hitachivantara) [reply .github/workflows/bootstrap-image.yml:36]: ok sure

File: settings.xml
Comment (mayur-hitachivantara) [reply settings.xml:1]: Ok

**Review by cardosov — APPROVED** [review]

**Review by vganpisetty — APPROVED** [review]

**Pull request merged**